web/api/errors.go

```diff
@@ -1,25 +1,27 @@
 package api
 
-//Error - http level error type
+// Error - http level error type.
 type Error string
 
-//Error - implementation of std.Error protocol
+// Error - implementation of std.Error protocol.
 func (e Error) Error() string { return string(e) }
 
 const (
-	//ErrorWrongInput request data is corrupted
+	// ErrorWrongInput request data is corrupted.
```

Comment:
I wouldn't dismiss auxiliaries. To my mind, comment look much clearer when they are grammatically correct sentences. E.g, ErrorWrongInput **occurs when/is when** request data is corrupted, ErrorRequestSignature **is a** HMAC request signature error etc.